.classpath

```diff
@@ -11,10 +12,73 @@
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-jms-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-test-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-aspects-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-messaging-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-webflux-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-webmvc-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-websocket-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-context-support-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-orm-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-web-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-jdbc-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-context-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-tx-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-oxm-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-aop-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-beans-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-expression-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-core-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-context-indexer-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-instrument-5.2.3.RELEASE.jar"/>
+	<classpathentry kind="lib" path="/Users/jasonwahjudi/Downloads/spring-framework-5.2.3.RELEASE/libs/spring-jcl-5.2.3.RELEASE.jar"/>
```

Comment:
Can you replace this whole classpath file with the one that was there previously? The same one on master. You can find it [here](https://github.com/usfcs-perf-eng-s20/heroku-project-favs/blob/master/.classpath). And going forward if you don't want any changes to this classpath file to be committed, you can add it to the .gitignore file